sshu/daml_sshu.cc

```diff
@@ -0,0 +1,85 @@
+#include "eckit/config/YAMLConfiguration.h"
+#include "./eckit/filesystem/PathName.h"
```

Comment:
why is one `eckit` and the other `./eckit`?

Reply:
ha! good catch, not sure why this even works!